pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -93,6 +94,9 @@ func (p *ConfigurationEditFlags) addSharedFlags(command *cobra.Command) {
 	command.Flags().IntVar(&p.MaxScale, "scale-max", 0, "Maximum number of replicas.")
 	p.markFlagMakesRevision("scale-max")
 
+	command.Flags().StringVar(&p.ScaleMetric, "scale-metric", "", "Metric the PodAutoscaler should scale on.")
```

Comment:
Can we add a few examples here, like `rps` or `concurrency` maybe? Just for better illustration.

Reply:
Wdyt? I'd like to highlight it's "name": `Name of metric to scale on by the PodAutoscaler.`

Reply:
Thanks, I just updated the flag usage. I also updated the usage for `--scale-target` flag. Does it read better now?